src/subsystems/shooter.py

```diff
@@ -62,7 +64,14 @@ def __init__(
 
         scheduler.registerSubsystem(self)
 
+        # SmartDashboard.putNumber("p", 3.724)
+        # SmartDashboard.putNumber("i", 0)
+        # SmartDashboard.putNumber("d", 0.05)
+
     def periodic(self):
+        # self.pitch_pid.setP(SmartDashboard.getNumber("p", 0))
+        # self.pitch_pid.setI(SmartDashboard.getNumber("i", 0))
+        # self.pitch_pid.setD(SmartDashboard.getNumber("d", 0))
```

Comment:
Can you just remove this commented stuff, the commented import at the top of the file, and the commented print at the bottom? Other than that, approved